src/MiniZinc.jl

```diff
@@ -71,13 +69,6 @@ function MOI.supports_constraint(
     return true
 end
 
-MOI.supports(::Model, ::MOI.NLPBlock) = true
-
-function MOI.set(model::Model, ::MOI.NLPBlock, data::MOI.NLPBlockData)
-    model.ext[:nlp_block] = data
-    return
-end
```

Comment:
I think we want to keep both approaches. Otherwise removing this is breaking.

My plan is that `@NL` macros will continue to create `NLPBlockData`, and the new nonlinear syntax in `@constraint` will make `ScalarNonlinearFunction`

Reply:
Let's discuss further, perhaps on a call? I don't feel like solver wrappers should have to handle two separate pathways like this, and I'm not sure why both are needed in MOI.